faithcomesbyhearing/dbp
# FCBH-983 Review dbp v4 docs

# Description
- Changed default version to `v4`
- Changed default format version to `json`
- Added redirection to github getting started

## Issue Link
Original Story: [FCBH-983](https://fullstacklabs.atlassian.net/browse/FCBH-983) 
Review Story: [FCBH-1373](https://fullstacklabs.atlassian.net/browse/FCBH-1373) 

## How Do I QA This
- Navigate to https://dbp.test/ and click on `Get Started` and verify you are redirected
- Click on `API -> about.getting_started` and verify you are redirected
- Test one random endpoint on https://dbp.test/docs/swagger/v4 and verify is working

## Checklist
